### DISCUSSION TITLE
Had ssl issue with bert-base-uncased where should I put it?

### DISCUSSION THREAD

**Discussion by meanin2** [lllyasviel/Fooocus#3336] [Q&A] [answered]
I had an SSL issue while fooocus was downloading the bert-base-uncased model.
I downloaded it manually. 
Where should I put it now? 
What should I name the file?
Thanks!

**(The download link that failed in terminal)**
https://cdn-lfs.huggingface.co/bert-base-uncased/68d45e234eb4a928074dfd868cead0219ab85354cc53d20e772753c6bb9169d3?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27model.safetensors%3B+filename%3D%22model.safetensors%22%3B&Expires=1721829453&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTcyMTgyOTQ1M319LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy5odWdnaW5nZmFjZS5jby9iZXJ0LWJhc2UtdW5jYXNlZC82OGQ0NWUyMzRlYjRhOTI4MDc0ZGZkODY4Y2VhZDAyMTlhYjg1MzU0Y2M1M2QyMGU3NzI3NTNjNmJiOTE2OWQzP3Jlc3BvbnNlLWNvbnRlbnQtZGlzcG9zaXRpb249KiJ9XX0_&Signature=BkzOmjo18c9OxKNKubHXrq6Oqgsxy0~mgL~ikHlWHM1pmg9O5ZHpoZrdlpQe~36eXHRb6NxiEiHbPZ1-DyRSN7JQjqEe1Rp1JVJOn1O65YfFlfaY3VVFdMBjCDQ36nArAAMUAHxU5sok1NKghXnatUkhQpybnieo29dUIIzR4tLLJokiJ1X6Xr~KR2w3BP1DpuNRaZB5uhc8C1U7gC4MAOmdQD2nkU54CoiPzsWzHH73LlGhjHsuWQJUr5C1Kw2DqhGlbMvrXzShlrwxVoHomlduKHl8ejyEpEEv~gfoHkvBhCuf-iWTDAwrvrUCySVHTW5SJoVmvl3gTmu1tp0P0g__

**Comment by mashb1t** [comment, marked as answer]
It's in your huggingface cache folder. Depending on your OS it can be in:
- Windows: `C:\Users\<username>\.cache\huggingface\hub\models--bert-base-uncased`
- Linux: `~/.cache/huggingface/hub/models--bert-base-uncased`

It's easiest if you delete the folder and let Fooocus auto-download the model again as the blob files in models--bert-base-uncased/blobs are cryptic.

**Reply by meanin2** [reply]
Thanks!
